release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -934,6 +934,17 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use short-time security credentials that are managed outside of the cluster. 
```

Comment:
I think `short-lived` or `short-term` are more commonly used in this context (vs `short-time`), or even `temporary security credentials`.

Reply:
Fixed with "Temporary"